msi-ec.c

```diff
@@ -724,8 +729,73 @@ static struct platform_driver msi_platform_driver = {
 	.driver = {
 		.name = MSI_DRIVER_NAME,
 	},
-	.probe	= msi_platform_probe,
-	.remove	= msi_platform_remove,
+	.probe = msi_platform_probe,
+	.remove = msi_platform_remove,
+};
+
+// ============================================================ //
+// Sysfs leds subsystem
```

Comment:
Great job! I don't know the existence of this subsystem. Could you add the documentation about how to use it in the README file?

Reply:
sure